config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+aMclSLfxhNA0UmyhyeU+x96T6j0aKeI3bPanMs4RurhiyATSph/JQxL5gZwU6ZekDo6Jb4i4jC5iMTlNNe/e/Pij9LVZ2DvJkKtkAbPEzy6Y+wIRMO4aVCzWqNsd4T6ZDNerkf0OugHicj4yadsjPBUHJT9UeV9tju2rswY6UDkx5L1IPoMxJzPNZSSpxP/ATLJyc9tGQr/3bW1mrVIix8zXRBzAbld96SUlQiSkVpLCcGN96IfkYvhsol1HTLtzq4yABL4RUCO8e0VPtr04v4WJSU+BSIljNWtwmcMjbVzmweTo8Cx2YBNnrle3H/haR2I9OQUe/fUtm9awLuCcBmN0WB9y3WC+Z/M0MwHBG++IWeLNOs6emibhpvB55Fh2z4mfICsT+FalvdkA/ammmoYW0QNxYmYX9u+t--7LRf03Skibyy7fv1--6ExwLt521KnLHP9SZO/Zww==
```

Comment:
gitignore this file

Reply:
is it copy this to my buttom part of my gitignore file?

Reply:
after adding this file, i need to push again right?

Reply:
after i push, terminal wrote everything is up-to-date. How can i update again in github?

Reply:
after adding file you have to commit it 
try to follow  this
`
git add .
git comit --amend --no-edit
git push origin init_project -f 
`

Reply:
if this is a copy file, you don't need to  gitignore it

Reply:
i just push it already. Can u commit which part I still need to correct?